src/platforms/dialogflow/google/GoogleAssistantPlatform.ts

```diff
@@ -51,6 +51,7 @@ import {
   SessionEntity,
   Suggestions,
   Table,
+  Telephony,
```

Comment:
This are not google assistant directives, but instead a whole other platform. Maybe it makes sense to work on it similarly to the way we worked the Dialogflow facebook platform? 

https://github.com/VoxaAI/voxa/tree/master/src/platforms/dialogflow/facebook